modules/@apostrophecms/ui/ui/apos/components/AposContextMenu.vue

```diff
@@ -138,7 +145,7 @@ export default {
         this.unbind();
       }
     },
-    buttonClicked() {
+    buttonClicked($event) {
```

Comment:
It doesn't look like `$event` is used. Does that mean the addition in `AposButton` is not needed?

Reply:
removed, unneeded